test/unit/provisioning/singleGroupConfigurationMode-test.js

```diff
@@ -0,0 +1,348 @@
+/*
```

Comment:
Is test/unit/provisioning/singleGroupConfigurationMode-test.js a "parallel" of some existing service-based test?

Reply:
Yes, It is parallel of test/unit/provisioning/singleConfigurationMode-test.js